### PR TITLE
Using Bigtable ReadRowsRequest instead of Scan for CloudBigtableIO.

### PR DESCRIPTION
**Pull request by sduskis**
This removes a use of HBase's ProtobufUtils which uses HBase protos under the cover, which requires old versions of protobuf to work.  Removing uses of ProtobufUtils will allow us to upgrade grpc and protobuf versions.

### CODE REVIEW

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java [hunk 1]
```diff
@@ -92,6 +90,17 @@ public Builder() {
      */
     public Builder withScan(Scan scan) {
       this.scan = scan;
+      this.request = null;
+      return this;
+    }
+
+    /**
+     * Specifies the {@link ReadRowsRequest} that will be used to filter the table.
+     * @param scan The {@link ReadRowsRequest} to add to the configuration.
```
Comment (garye) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: @param scan -> @param request.

File: bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java [hunk 2]
```diff
@@ -92,6 +90,17 @@ public Builder() {
...
+     * @return The {@link CloudBigtableScanConfiguration.Builder} for chaining convenience.
+     */
+    public Builder withRequest(ReadRowsRequest request) {
+      this.request = request;
```
Comment (garye) [inline bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableScanConfiguration.java]: Should this.scan be set to null to mimic the behavior of withScan()?

**Comment by sduskis** [comment]
All comments addressed.  PTAL.

**Comment by garye** [comment]
LGTM

**Pull request merged**